src/test/java/service/ServiceTests.java

```diff
@@ -0,0 +1,183 @@
+package service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import dao.FruitStorageDao;
+import dao.FruitStorageDaoImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import model.Fruit;
+import model.InputDataType;
+import model.Operation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import service.impl.GenerateReportImpl;
+import service.impl.InputDataResolverImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.TransactionOperationImpl;
+import service.impl.WriterServiceImpl;
+
+class ServiceTests {
+    private static final String TEST_FILE = "src/main/resources/test.csv";
+    private ReaderService reader;
+    private WriterService writer;
+    private InputDataResolver resolver;
+    private TransactionOperation transactionOperation;
+    private GenerateReport report;
+    private FruitStorageDao fruitStorageDao;
+    private Path testFilePath;
```

Comment:
Unit-test class should test only one implementation. And your test classes tree should have absolutely same structure as main package in order to make understanding of what it tested easier.
Please separate and place your tests properly.
e.g.
we have main/java/service/impl/WriterServiceImpl.java
that means that tests for it are expected as test/java/service/impl/WriterServiceImplTest.java